BEIMA.Client/cypress/integration/Router.spec.js

```diff
@@ -0,0 +1,23 @@
+/// <reference types="cypress" />
+
+describe("Router Doesn't Redirect On Valid Pages", () =>{
+  it('Visits Root', () =>{
+    cy.visit('http://localhost:3000')
+    cy.url().should('eq','http://localhost:3000/')
```

Comment:
We should add another test that puts random text in the url and verify it routes to the home page